docs/_writing-tests/index.md

```diff
@@ -11,43 +11,47 @@ There's also a load of [general guidelines][] that apply to all tests.
 
 ## Test Type
 
-There are four main test types:
+Tests in this project use a few different approaches to verify expected
+behavior. The tests can be classified based on the way they express
+expectations:
 
-* [Reftests][] should be used to test rendering and layout. They
-  consist of two or more pages with assertions as to whether they
-  render identically or not.
+* Rendering tests should be used to verify that the browser graphically
+  displays pages as expected. See the [rendering test guidelines][rendering]
+  for tips on how to write great rendering tests. There are a few different
+  ways to write rendering tests:
 
-* [testharness.js][] tests should be used (where
-  possible!) for testing everything else. They are built with the
-  testharness.js unit testing framework, and consist of assertions
-  written in JavaScript.
+  * [Reftests][] should be used to test rendering and layout. They consist of
+    two or more pages with assertions as to whether they render identically or
+    not.
 
-* [Visual tests][visual] should be used for checking rendering where
-  there is a large number of conforming renderings such that reftests
-  are impractical. They consist of a page that renders to final state
-  at which point a screenshot can be taken and compared to an expected
-  rendering for that user agent on that platform.
+  * [Visual tests][visual] should be used for checking rendering where there is
```

Comment:
At this point, there's only marginal value in having visual tests (they're practically identical to manual tests, given nobody has automation to handle them nowadays). We probably don't want to talk about screenshots here, but rather note they're essentially a subset of manual tests.

Reply:
It sounds like Visual Tests can be removed from these overviews completely, and a reference instead added to the "Manual Tests" page. Would you agree?

Reply:
So I have some concern about it becoming hard to find out what `-visual` tests are (especially once we bulk rename CSS tests in #5381), even though really they're just a special case of manual tests at this point.